mysql-test/suite/innodb/t/encryption_force.test

```diff
@@ -0,0 +1,107 @@
+-- source include/have_innodb.inc
```

Comment:
I'd name "percona_force_encryption" so that it's immediately visible in the runs as a Percona feature testcase